pkg/controller/account/ec2.go

```diff
@@ -88,8 +88,10 @@ func (r *ReconcileAccount) InitializeRegion(
 	reqLogger.Info("initializing region", "region", region)
 
 	// Attempt to clean the region from any hanging resources
-	cleaned, err := cleanRegion(awsClient, reqLogger)
+	cleaned, err := cleanRegion(awsClient, reqLogger, account.Name)
 	if err != nil {
+		cleanErr := fmt.Sprintf("Error", err.Error(), "account", account.Name)
```

Comment:
@ArielLima I suggest having a more descriptive msg than Error here.

The previous written msg would work fine IMO, and then you could append the aws err string as you're doing